charts/prometheus/templates/clusterrolebinding.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: {{ template "prometheus.serviceAccountName.server" . }}
-{{ include "prometheus.namespace" . | indent 4 }}
+    namespace: {{ template "prometheus.namespace" . }}
```

Comment:
This is not relevant. Please keep it as-is. Plus, the official document recommend to use `{{ include ... }}` not `{{ template ... }}`.  See:

https://helm.sh/docs/chart_template_guide/named_templates/#the-include-function
https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-include-function

Reply:
Ok, revert and keep it as-is.

Reply:
Done, but a slightly clumsy solution.

Split the namespace fix if you want, do you?